### PR TITLE
Rune2code

### PR DESCRIPTION
**Pull request by sago35**
Added function Code () for reverse lookup.

### CODE REVIEW

File: convert.go [hunk 1]
```diff
@@ -6900,10 +6900,36 @@ var codeToRune = map[int]rune{
...
+// Rune retrieves the rune corresponding to code
 func Rune(code int) (rune, error) {
 	r, ok := codeToRune[code]
 	if ok {
 		return r, nil
 	}
 	return 0, errors.New("value out of range")
 }
+
+// Code retrieves the code corresponding to rune
```
[the remark this answers is not in the document]
Comment (sago35) [reply convert.go]: No, Code(r rune) retrieves jisx0208 code.

[the remark this answers is not in the document]
Comment (hajimehoshi) [reply convert.go]: Ah ok thanks

**Comment by sago35** [comment]
Thx.

File: README.md [hunk 2]
```diff
@@ -8,4 +8,10 @@ if err != nil {
        panic(err)
 }
 fmt.Printf("%c\n", ch) // あ
+
+r, err := jisx0208.Rune(ch)
+if err != nil {
+       panic(err)
+}
+fmt.Printf("%c\n", r) //
```
Comment (hajimehoshi) [inline README.md]: Oh, could you complete the comment?

Comment (sago35) [reply README.md]: OK. I will add a comment from now.

**Pull request merged**